Packs/FreshDesk/Integrations/FreshDesk/FreshDesk_test.py

```diff
@@ -0,0 +1,98 @@
+import demistomock as demisto
+import io
+import json
+
+
+MOCK_PARAMS = {
+    'credentials': {
+        'identifier': 'TEST',
+        'password': 'TEST'
+    },
+    'url': 'https://MOCK_URL',
+    'maxFetch': '14',
+}
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+# The unitest timestamps where taken according to the remote machine time zone (UTC).
+def test_fetch_incidents_no_pagination(mocker, requests_mock):
+    """Unit test
+    Given
+    - fetch incidents command where no pagination is needed
```

Comment:
Lets split this UT:
1 - there are 10 incidents in the system, the customer requested 5.
2 - there are 10 incidents in the system, the customer requested 15.

Reply:
Done